config/config.go

```diff
@@ -55,24 +56,25 @@ func Parse(rawConfig []byte) (types.Config, report.Report, error) {
 		}
 
 		return config, report.ReportFromError(ErrDeprecated, report.EntryDeprecated), nil
+	case types.MaxVersion:
+		return ParseFromLatest(rawConfig)
```

Comment:
It probably doesn't make sense to futz with this anymore. I think the original idea was that we were looking for the "version 1" config and tried parsing with the latest otherwise.